airflow/contrib/operators/sqoop_operator.py

```diff
@@ -156,9 +176,15 @@ def execute(self, context):
                 input_lines_terminated_by=self.input_lines_terminated_by,
                 input_optionally_enclosed_by=self.input_optionally_enclosed_by,
                 batch=self.batch,
-                relaxed_isolation=self.relaxed_isolation)
+                relaxed_isolation=self.relaxed_isolation,
+                extra_export_options=self.extra_export_options)
         elif self.cmd_type == 'import':
-            if not self.table:
+            # add create hcatalog table to extra import options if option passed
+            # if new params are added to constructor can pass them in here so don't modify sqoop_hook for each param
+            if self.create_hcatalog_table:
```

Comment:
this comment says it all but we should start passing in these new options we add to the SqoopOperator constructor to the extra args so don't change the SqoopHook so often

Reply:
👍